packages/components/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 -   Tweak `WordPressComponent` type so `selector` property is optional ([#49960](https://github.com/WordPress/gutenberg/pull/49960)).
 -   Update `Modal` appearance on small screens ([#50039](https://github.com/WordPress/gutenberg/pull/50039)).
 -   Update the framer motion dependency to the latest version `10.11.6` ([#49822](https://github.com/WordPress/gutenberg/pull/49822)).
+-   The undocumented `onHTMLDrop` prop from the `<MediaPlaceholder>` component is deprecated ([#49673](https://github.com/WordPress/gutenberg/pull/49673)).
```

Comment:
I believe that `MediaPlaceholder` is not part of `@wordpress/components` package — should this entry be moved to the `@block-editor` CHANGELOG ?

Instead, for the `@wordpress/components` package, we should probably document the removal of `event.persist` ?

Reply:
Oops! Yep, I misread my own code 😵 . Thanks!